package.json

```diff
@@ -37,16 +43,20 @@
   },
   "main": "src/index.js",
   "scripts": {
-    "sst": "mocha test/acceptance/index.test.js --timeout 15000",
-    "loc": "sloc src/ examples/",
-    "test": "npm run lint",
-    "lint": "eslint .",
-    "lint:fix": "eslint . --fix",
-    "build-dev": "webpack --config webpack/webpack.config.js",
     "build-all": "webpack --config webpack/webpack.config.js && webpack --config webpack/webpack.min.config.js",
-    "build:watch": "webpack -w --config webpack/webpack.config.js",
-    "build-docs": "rm -rf docs/public; jsdoc --configure config/jsdoc/public-conf.json",
+    "build-dev": "webpack --config webpack/webpack.config.js",
     "build-docs-all": "rm -rf docs/all; jsdoc --configure config/jsdoc/all-conf.json",
-    "ghpublish": "git checkout gh-pages && git merge master && yarn build-dev && yarn build-docs && git commit -a -m \"Auto generated gh-pages\" && git push && git checkout master"
+    "build-docs": "rm -rf docs/public; jsdoc --configure config/jsdoc/public-conf.json",
+    "build:watch": "webpack -w --config webpack/webpack.config.js",
+    "build": "webpack --config webpack/webpack.config.js && webpack --config webpack/webpack.min.config.js",
+    "docs:all": "rm -rf docs/all; jsdoc --configure config/jsdoc/all-conf.json",
+    "docs": "rm -rf docs/public; jsdoc --configure config/jsdoc/public-conf.json",
+    "ghpublish": "git checkout gh-pages && git merge master && yarn build-dev && yarn build-docs && git commit -a -m \"Auto generated gh-pages\" && git push && git checkout master",
+    "lint:fix": "eslint . --fix",
+    "lint": "eslint .",
+    "loc": "sloc src/ examples/",
+    "publish": "git checkout gh-pages && git merge master && npm run build && npm run build-doc && git commit -a -m \"Auto generated gh-pages\" && git push && git checkout master",
```

Comment:
Why is this command duplicated (`ghpublish`)?

Reply:
`ghpublish` is the good one. I changed in other PR, `publish` to `ghpublish` because `publish` is a built-in npm option.